server/odc-service/src/main/java/com/oceanbase/odc/metadb/config/ConfigEntity.java

```diff
@@ -27,26 +32,35 @@
  * @Description: []
  */
 @Data
+@MappedSuperclass
 public class ConfigEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
     /**
      * configuration key
      */
     @NotBlank
+    @Column(name = "`key`")
```

Comment:
bad practive for identify quote char here,
can we set spring.jpa.properties.hibernate.dialect in application.yml to workaround ?

Reply:
This is used to bypass mysql reserved words.
It's true that hibernate have these configuration, such as `globally_quoted_identifier `, but the impact of these changes is far too broad.Custom implementations may be required to adapt to MySQL and H2 compatibility, and this could also lead to issues such as case sensitivity.